SU2_CFD/src/numerics/CNumerics.cpp

```diff
@@ -1758,40 +1756,61 @@ void CNumerics::GetPrimitive2Conservative (const su2double *val_Mean_PrimVar,
   }
 }
 
-
-
 void CNumerics::CreateBasis(const su2double *val_Normal) {
+  
```

Comment:
This function is a prime candidate to be phased out using the Geometry toolboxes.